libs/stats/odc/stats/_cli_publish_tasks.py

```diff
@@ -32,8 +34,9 @@ def get_tasks(cache_file):
             }
         messages.append(message)
         counter += 1
-        # ToDo: add a check that ensures a message has been delivered
+        print(counter)
```

Comment:
debug output that should not be here?

Reply:
Yeah, it slipped through.  It's removed now.